### ISSUE TITLE
"how to use murmur" tutorial/FAQ

### ISSUE THREAD

**Issue by kmahar** [haystack/murmur#66]
I added a few of my friends to a mailing list so we can get some more people playing around with it. They have been pretty confused and asking me a lot of questions about how it works. So before rolling it out to a lot of testers we might want to make an FAQ or brief tutorial explaining the features and how to use them.

**Comment by karger** [comment]
We might also want to understand what they are confused about and why. 
Ideally, the way to use it will be clear from using it, without need to
resort to documentation.

**Comment by kmahar** [comment]
Yeah. In particular, they were confused about the "purpose" of upvoting,
and they didn't realize that they could still view muted threads on the
murmur site.
On Thu, Feb 18, 2016 at 2:15 PM David Karger <notifications@github.com>
wrote:

> We might also want to understand what they are confused about and why.
> Ideally, the way to use it will be clear from using it, without need to
> resort to documentation.
>
> On 2/18/2016 1:52 PM, Kaitlin Mahar wrote:
> >
> > I added a few of my friends to a mailing list so we can get some more
> > people playing around with it. They have been pretty confused and
> > asking me a lot of questions about how it works. So before rolling it
> > out to a lot of testers we might want to make an FAQ or brief tutorial
> > explaining the features and how to use them.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/haystack/murmur/issues/66>.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/haystack/murmur/issues/66#issuecomment-185869813>.
>
-- 
Kaitlin Mahar
Massachusetts Institute of Technology | Class of 2016
Department of Electrical Engineering and Computer Science
kmahar@mit.edu

**Comment by karger** [comment]
Viewing muted threads on the site is useful.  But it should also be
possible to hide them on the site, to reduce clutter.

I guess the confusion about upvoting is that it is not typically
associated with email. If they were on the site, they would understand
perfectly well what upvoting was for, right?

Is there another term that would make it more understandable? 
Recommend?  Share?   Like?

**Comment by amyxzhang** [comment]
I could see the confusion because right now liking an email doesn't really
do that much. Of course in the future, you'd be able to do things based on
that signal like only see emails that have received some # of likes, etc.
or get the email forwarded to more people. As a first thing, I'd like the
poster of the email to get an email letting them know when people have
liked their email - this would be a nice little bit of encouragement I
think. There's already an issue posted for that feature.

On Fri, Feb 19, 2016 at 12:48 AM, David Karger <notifications@github.com>
wrote:

> Viewing muted threads on the site is useful. But it should also be
> possible to hide them on the site, to reduce clutter.
>
> I guess the confusion about upvoting is that it is not typically
> associated with email. If they were on the site, they would understand
> perfectly well what upvoting was for, right?
>
> Is there another term that would make it more understandable?
> Recommend? Share? Like?
>
> On 2/18/2016 2:28 PM, Kaitlin Mahar wrote:
> > Yeah. In particular, they were confused about the "purpose" of upvoting,
> > and they didn't realize that they could still view muted threads on the
> > murmur site.
> > On Thu, Feb 18, 2016 at 2:15 PM David Karger <notifications@github.com>
> > wrote:
> >
> > > We might also want to understand what they are confused about and why.
> > > Ideally, the way to use it will be clear from using it, without need to
> > > resort to documentation.
> > >
> > > On 2/18/2016 1:52 PM, Kaitlin Mahar wrote:
> > > >
> > > > I added a few of my friends to a mailing list so we can get some more
> > > > people playing around with it. They have been pretty confused and
> > > > asking me a lot of questions about how it works. So before rolling it
> > > > out to a lot of testers we might want to make an FAQ or brief
> tutorial
> > > > explaining the features and how to use them.
> > > >
> > > > —
> > > > Reply to this email directly or view it on GitHub
> > > > <https://github.com/haystack/murmur/issues/66>.
> > > >
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <https://github.com/haystack/murmur/issues/66#issuecomment-185869813>.
> > >
> > --
> > Kaitlin Mahar
> > Massachusetts Institute of Technology | Class of 2016
> > Department of Electrical Engineering and Computer Science
> > kmahar@mit.edu
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/haystack/murmur/issues/66#issuecomment-185878643>.
>
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/haystack/murmur/issues/66#issuecomment-186067173>.
>



-- 
Amy X. Zhang | Ph.D. student at MIT CSAIL | http://people.csail.mit.edu/axz
| @amyxzh
